backend/game/player/player.py

```diff
@@ -32,3 +36,9 @@ def update_energy_by(self, change_integer):
         self.energy += change_integer
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
+
+    def add_keep_card(self, card):
+        self.card_on_hand[card.name] = card.effect
+
+    def remove_keep_card(self, card):
+        self.card_on_hand.pop(card.name)
```

Comment:
If changing to a list, this method will have to be redone

Reply:
This is redone in #68